spec/android_spec.rb

```diff
@@ -223,6 +223,17 @@
           allow(Open3).to receive(:capture3) { [out, '', $STATUS_ZERO] }
           expect(device.ip_address).to eq('192.168.1.1')
         end
+
+        it 'will not crash if Wifi is not enabled' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.